### ISSUE TITLE
Update to Bootstrap 5

### ISSUE THREAD

**Issue by corylown** [sul-dlss/exhibits#2513]
Work in progress here: https://github.com/sul-dlss/exhibits/compare/bs5?expand=1

Depends on:
- [x] New release of Spotlight with https://github.com/projectblacklight/spotlight/commit/b0873d3a9bc0f140682cc6cad5eac0ca6eaa53f6
- [x] Updating bootstrap_form to `~> 5.4` in Exhibits

Gotchas: blacklight_range_limit does not work with Bootstrap 5:
* https://github.com/projectblacklight/blacklight_range_limit/issues/177
* https://github.com/projectblacklight/blacklight_range_limit/issues/210
* https://github.com/projectblacklight/blacklight_range_limit/issues/212
* https://github.com/projectblacklight/blacklight_range_limit/issues/231 

There are accessibility concerns with blacklight_range_limit anyway, and based on discussions with @caaster a more limited date facet UI may be acceptable. Here's what we did for stanford-arclight for now:
https://github.com/sul-dlss/stanford-arclight/commit/c9264c80152e388f599711a2663e18636e12c81c

TODO:

- [x] Search tips modal has an empty button on the lower right
- [x] More details modal has 3 scroll bars
- [x] More details layout and lower button bar layout needs adjustment
- [x] BL Range Limit: turn off JS or update to new version once released
- [x] SUL logo has visible text over logo
- [x] IIIF logo should to the lower right side of the thumbnail (not lower left)
- [x] BL gallery selected layout is not displayed as active
- [x] Check that active style is applied to grouped by collection results
- [x] Tags are displaying as buttons and not links
- [x] Search bar is missing for "Find items in all exhibits"
- [x] Drop down next to site wide search should not have rounded corners on the right side
- [x] Exhibit search field select dropdown is too narrow
- [x] Test Parker specific features
- [x] Parker more details modal has an extra scrollbar
- [x] exhibit search without a search field dropdown should have rounded corners on left side. (this appears to be a blacklight bug: https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/search_bar_component.html.erb#L25-L30 )
- [x] Page width is off -- scrolls off on the right side
- [x] indexing status modal has horizontal scrollbar
- [x] Top nav & bulk actions menus don't work
- [x] browse category search bar and button should line up
- [x] browse category search bar should not look terrible when there's a search term present
